code/modules/vending/security.dm

```diff
@@ -21,7 +21,8 @@
 				   /obj/item/clothing/head/helmet/blueshirt = 1,
 				   /obj/item/clothing/suit/armor/vest/blueshirt = 1,
 				   /obj/item/clothing/gloves/tackler = 5,
-				   /obj/item/grenade/stingbang = 1)
+				   /obj/item/grenade/stingbang = 1,
+				   /obj/item/watertank/security = 1)
```

Comment:
Is this only available in the vending machine, and only one? I'd suggest throwing one in the armory.

Reply:
Throwing one in the armory would be a map-edit and I don't see myself editing every map individually just to add a pepperspray tank in each armory. Call me lazy but I'd rather have it available in vendors available to everyone.
As for the number of tanks in each vendor, I don't really know. What do you suggest? Would having two instead of one be good?